src/spaceone/inventory/manager/container_instances/container_manager.py

```diff
@@ -125,3 +130,21 @@ def set_volumes_detail_info(volume, containers):
                             'container_name': container['name']
                         })
                         return
+
+    @staticmethod
+    def convert_start_time_iso861_to_datetime(container):
+        _start_time = None
+        if container['instance_view'] is not None:
+            _start_time = container['instance_view']['current_state']['start_time']
+            if _start_time is not None:
+                _start_time = datetime_to_iso8601(_start_time)
```

Comment:
It is worth considering the cases in which an IndexError can be raised.
In case of `container['instance_view']['current_state']['start_time']` , an unexpected error (ex. IndexError) may occur.

Accordingly, if it is a dictionary type and has a key, it is recommended to access it as `container['key']`, and if not, it is recommended to declare a default value such as `container.get('instance_view', {})` .

Other than this, there seems to be nothing wrong. : )

Reply:
@stat-kwon Thanks for review.
I included your suggestions for the code.

Reply:
@stat-kwon Thanks for your detailed review.
I didn't think about that. I included your suggestions for the code.